app/controllers/default_bills_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class DefaultBillsController < ApplicationController
+  before_action :authorize_request
+  before_action :find_user, except: %i[show destroy]
+  before_action :find_default_bill, except: %i[index create]
+
+  def index
+    default_bills = DefaultBill.where(user_id: @user.id).all
+
+    render json: default_bills, status: :ok
```

Comment:
Aqui tbm pode ter um problema de segurança, o mais seguro seria usar tipo:
```suggestion
    render json: current_user.default_bills, status: :ok
```